ReadersAndroidJetpack/app/src/main/java/com/kontinua/readersandroidjetpack/views/NavbarView.kt

```diff
@@ -111,8 +111,12 @@ fun ChapterSidebar(onClose: () -> Unit, onButtonClick: () -> Unit, navbarManager
         verticalArrangement = Arrangement.spacedBy(10.dp)
     ) {
         WorkbookButton(onClick = onButtonClick)
-        for (chapter in chapters){
-            Text("Chapter ${chapter.chapNum}: ${chapter.title}", modifier = Modifier.clickable {
+        for (i in chapters.indices){
+            val chapter = chapters[i]
+            val bgColor = if(i == navbarManager.currentChapterIndex) Color.LightGray else Color.Transparent
+            Text("Chapter ${chapter.chapNum}: ${chapter.title}", modifier = Modifier
```

Comment:
generally would recommend using localization from the get-go. (so, probably a string resource lookup.) and formatting numbers using an appropriate formatter vs raw string interpolation.